vaccine_feed_ingest/stages/enrichment.py

```diff
@@ -96,3 +97,22 @@ def _add_provider_from_name(loc: location.NormalizedLocation) -> None:
 
     if not loc.parent_organization:
         loc.parent_organization = location.Organization(id=provider_authority)
+
+
+def _add_source_link(loc: location.NormalizedLocation) -> None:
+    """Add source link from source if missing"""
+    if not loc.source:
+        return
+
+    if not loc.source.source or not loc.source.id:
+        return
+
+    existing_links = _generate_link_map(loc)
+
+    if str(loc.source.source) in existing_links:
+        return
```

Comment:
Similar question to the other PR - should we check that this has the "right" id?

Reply:
If the normalizer added a different id then i assume they had a good reason at this stage.

If we want to do "validation" then we should do that in the schema.